src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.cs

```diff
@@ -180,10 +180,104 @@ public void Stop()
             NativeWebRTC.Stop(Handle).ThrowIfFailed("Failed to stop the WebRTC");
         }
 
+        /// <summary>
+        /// Creates SDP offer to start a new WebRTC connection to a remote peer and set it to local.
+        /// </summary>
+        /// <remarks>
+        /// This API is helper method to create offer, set local description in one shot.<br/>
+        /// The WebRTC must be in the <see cref="WebRTCState.Negotiating"/>
+        /// </remarks>
+        /// <returns>The SDP offer.</returns>
+        /// <exception cref="InvalidOperationException">The WebRTC is not in the valid state.</exception>
+        /// <exception cref="ObjectDisposedException">The WebRTC has already been disposed.</exception>
+        /// <seealso cref="CreateOffer()"/>
+        /// <seealso cref="CreateOffer(Bundle)"/>
+        /// <seealso cref="SetLocalDescription"/>
+        /// <since_tizen> 9 </since_tizen>
+        public string CreateSetOffer() => CreateSetOffer(null);
```

Comment:
Considering that it's behavior is CreateOffer + SetLocalDescription(),
how about CreateOfferAndSet(Local)() for it's naming?

(By the way, I don't think it's not that useful for the previous webRTC user, they can just call two functions sequentially to have same effect.)

Reply:
I think it's useful because it can remove useless method call.

Reply:
Okay, then, let's think out the proper name of it.